django/template/backends/django.py

```diff
@@ -24,6 +25,10 @@ def __init__(self, params):
         libraries = options.get("libraries", {})
         options["libraries"] = self.get_templatetag_libraries(libraries)
         super().__init__(params)
+        if "" in self.dirs:
+            logger = logging.getLogger("django.template")
+            logger.warning("An empty string is not a valid template directory")
+            self.dirs = filter(None, self.dirs)
```

Comment:
I would rather restore the previous behavior and ignore empty directories in `get_template_directories()`:
```python
def get_template_directories():
    # Iterate through each template backend and find
    # any template_loader that has a 'get_dirs' method.
    # Collect the directories, filtering out Django templates.
    cwd = Path.cwd()
    items = set()
    for backend in engines.all():
        if not isinstance(backend, DjangoTemplates):
            continue

        items.update(cwd / to_path(dir) for dir in backend.engine.dirs if dir)
        ...
```

Reply:
Fully agree. PR updated.